@kiva/kv-components/vue/KvSideSheet.vue

```diff
@@ -15,6 +15,7 @@
 			:class="{
 				'tw-w-0 tw-delay-200 tw-opacity-0': !open,
 				'lg:tw-w-1/2 tw-w-full tw-opacity-full': open,
+				'tw-h-screen': stickyControls,
```

Comment:
Will this work with the website header?

Reply:
Should have min height?

Reply:
I think we have two options, both somewhat complex and needing to watch for scrolling to change the UI (like we do for the home page tiles on scroll):
1. Add a prop to this component for navigation height, have the side sheet be full view height minus the navigation height, watch on scroll to change the height as you scroll, so the "full height" side sheet allows room for the navigation -> this is probably the cleanest
2. Have the controls stick to the bottom of the screen until you scroll down past the controls (basically when you scroll past the height of the side sheet)

Reply:
Different heights for mobile and desktop btw.